addons/takistan/CfgFactionClasses.hpp

```diff
@@ -0,0 +1,43 @@
+class CfgFactionClasses {
+    class TACU_Takistan_TNA {
+        displayName = "Takistan National Army";
+        side = 2;
+        priority = 4;
+    };
+
+    class TACU_Takistan_TNP {
+        displayName = "Takistan National Police";
+        side = 2;
+        priority = 4;
+    };
+    
+    class TACU_Takistan_Tehrik {
+        displayName = "Tehrik-I-Taliban Takistan";
+        side = 0;
+        priority = 4;
+    };
+};
+
+class CfgEditorSubcategories {
+    class TACU_Takistan_EdSubCat_TakistanSF {
+        displayName = "Men (Takistani SF)";
+    };
+    class TACU_Takistan_EdSubCat_Officer {
+        displayName = "Men (Officers)";
+    };
+    class TACU_Takistan_EdSubCat_Regular {
+        displayName = "Men (Regular)";
+    };
+    class TACU_Takistan_EdSubCat_Commandos {
+        displayName = "Men (Commandos)";
+    };
+    class TACU_Takistan_EdSubCat_Police {
+        displayName = "Men (Officers)";
+    };
```

Comment:
```suggestion

```
`Men (Officers)` already exists from `TACU_Takistan_EdSubCat_Officer`, so no need for an extra one with the same name.